released/packages/coq-tlc/coq-tlc.20210316/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "arthur.chargueraud@inria.fr"
+
+homepage: "https://github.com/charguer/tlc"
+dev-repo: "git+https://github.com/charguer/tlc.git"
+bug-reports: "https://github.com/charguer/tlc/issues"
+license: "MIT"
+
+synopsis: "TLC: A Library for Classical Coq"
+description: """
+Provides an alternative to the core of the Coq standard library, using classic definitions.
+"""
+
+build: [make "-j%{jobs}%"]
+install: [make "install"]
+depends: [
+  "coq" { >= "8.12" }
+]
+
+tags: [
+  "category:Miscellaneous/Coq Extensions"
+  "date:2021-03-16"
+  "keyword:library"
+  "keyword:classic"
```

Comment:
Note that TLC is the only development using `keyword:library`. Does it carry a special meaning or should we just remove it? Also, TLC is the only development using `keyword:classic`; other developments are using `keyword:classical logic`. See https://coq.inria.fr/opam/www/

Reply:
Hi Guillaume, sure, I have no idea why this keyword is there. Feel free to remove it. I will also remove it upstream, and change `classic` to `classical logic`. Thanks!